package.json

```diff
@@ -32,7 +32,6 @@
     "lint": "yarn lint:javascript && yarn lint:stylelint",
     "lint:javascript": "eslint --ext .jsx --ext .js .",
     "lint:stylelint": "stylelint './src/**/*.scss' --syntax scss --ignorePath .gitignore --custom-formatter ./config/stylelint/sassMsgFormatter.js",
-    "prepare": "yarn build",
```

Comment:
Thanks for re-adding this. It was lost in an automated "update w/ master" merge somehow.